src/Android/Services/CryptoPrimitiveService.cs

```diff
@@ -33,5 +35,20 @@ public byte[] Pbkdf2(byte[] password, byte[] salt, CryptoHashAlgorithm algorithm
             generator.Init(password, salt, iterations);
             return ((KeyParameter)generator.GenerateDerivedMacParameters(keySize)).GetKey();
         }
+
+        public byte[] Argon2id(byte[] password, byte[] salt, int iterations, int memory, int parallelism)
+        {
+            JavaSystem.LoadLibrary("argon2");
+            // TODO: Do we need to pass this in somehow like PBKDF2, based on an algorithm?
+            int keySize = 32;
+            var key = new byte[keySize];
+            argon2id_hash_raw(iterations, memory, parallelism,
+                password, password.Length, password, salt.Length, key, key.Length);
```

Comment:
I think it should be `salt` instead of `password` on the 6th parameter

Reply:
Yeah, I noticed this too in my other kdfConfiguration PR, not sure how this went unnoticed, but it should definitely be salt.

Reply:
Fixed.